### ISSUE TITLE
  [ENHANCEMENT] LCD Menu - Settings - Temperature, non-zero default values (or even better, Most Recently Used values)

### ISSUE THREAD

**Issue by JackBender** [prusa3d/Prusa-Firmware#2706]
**TL;DR:**
When using [LCD Menu - Settings - Temperature], sensible default values or Most Recently Used values would be a nice touch.

.

As far as I can tell, with the recent and current MK3S firmwares, you are greeted with the following display when you get in [LCD Menu - Settings - Temperature]:
> Nozzle:        0
> Bed:             0
> Fan speed:   0

When the user wants to edit one of these values, the value remains at 0 and the user has to frantically rotate the LCD knob to dial in the desired value.

It would be easier if the target values were the most recently used values instead of zeros.
The user would spend less time and effort rotating the LCD knob to set the desired values.

I guess most people using the menu [LCD Menu - Settings - Temperature] select
a nozzle temperature in the range [180-250],
a bed temperature in the range [50-100],
and a fan speed in the range [20-100].

In this case, average default values would make the life of pretty much everyone easier by significantly decreasing the number of LCD knob turns to perform.
Here is a **proposed example of average default values**:
> **Nozzle:     215**
> **Bed:            75**
> **Fan speed:  60**

.

_(As an added bonus, when the LCD knob is rotated rapidly, it would be more practical if the current value was increased or decreased by steps of 5 or 10 instead of steps of 1.)_

**Comment by JohnnyDeer** [comment]
Hi @JackBender , thank you for suggestion. We decided to not implement you´r idea at this moment. However, we have it recorded in the database in case we decide to use it one day.
